NuGet/Microsoft.ChakraCore.Debugger.nuspec

```diff
@@ -17,15 +17,15 @@
   </metadata>
 
   <files>
-    <file src="..\lib\Debugger.Service\ChakraDebugService.h" target="include" />
-    <file src="..\lib\Debugger.ProtocolHandler\ChakraDebugProtocolHandler.h" target="include" />
+    <file src="..\lib\Debugger.Service\ChakraDebugService.h" target="build\native\include" />
```

Comment:
>build [](start = 69, length = 5)

Why is this folder called "build"? It probably should be called "inc", "publicinc", or something like that.

Also, why do we need to specify "native"? Is it to differentiate from some javascript/typescript definition files?

Reply:
I'm adhering to the convention used by the ChakraCore package.